lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -365,10 +365,27 @@ describe('ContentSelector', () => {
       button.props().onClick();
     });
 
+    const bookPicker = wrapper.find('BookPicker');
+    assert.equal(bookPicker.prop('authToken'), fakeConfig.api.authToken);
+    assert.isTrue(wrapper.exists('BookPicker'));
+  });
+
+  it('submits selecting a book from VitalSource', () => {
```

Comment:
This test description is a little hard to parse.

Reply:
Agh. Indeed. Will rewrite.